scene/resources/capsule_shape_2d.cpp

```diff
@@ -97,7 +97,8 @@ void CapsuleShape2D::draw(const RID &p_to_rid, const Color &p_color) {
 }
 
 Rect2 CapsuleShape2D::get_rect() const {
-	return Rect2(0, 0, radius, height);
+	const Vector2 half_size = Vector2(get_radius(), get_height() * 0.5);
+	return Rect2(-half_size, half_size * 2.0);
```

Comment:
No need to use getters:
```suggestion
	const Vector2 half_size = Vector2(radius, height * 0.5);
	return Rect2(-half_size, half_size * 2.0);
```
Alternative:
```suggestion
	return Rect2(-radius, -height * 0.5, radius * 2.0, height);
```